.mailmap

```diff
@@ -157,6 +157,7 @@ Amit Kumar <dtu.amit@gmail.com>
 Amit Kumar <dtu.amit@gmail.com> <aktech@users.noreply.github.com>
 Amit Kumar <dtu.amit@gmail.com> <dtu.amit+gh@gmail.com>
 Amit Manchanda <amitdelhi1995@gmail.com>
+Amit Portnoy <amit.portnoy@gmail.com>
```

Comment:
The AUTHORS check fails with
```
This author is not included in the .mailmap file:
Amit Portnoy <1131991+amitport@users.noreply.github.com>
```
You should change the line above in `.mailmap` to
```
Amit Portnoy <amit.portnoy@gmail.com> Amit Portnoy <1131991+amitport@users.noreply.github.com>
```
That tells the `bin/mailmap_check.py` script that the second name/email is an alias for the first.